Source/Controllers/SubviewControllers/SPRuleFilterController.m

```diff
@@ -1182,6 +1183,13 @@ - (NSArray *)_compareTypesForColumn:(ColumnNode *)colNode
                     // the regex below is causing a crash:
                     // https://console.firebase.google.com/u/0/project/com-sequel-ace/crashlytics/app/ios:com.sequel-ace.sequel-ace/issues/8754587eadf991cbb11ccc83b9fe0b5b
                     // so I've added logging to RegexLite exception generation.
+
+                    SPLog(@"tip: %@", tip);
+                    CLS_LOG(@"tip: %@", tip);
+
+                    // FIXME: oh ... is the regex is wrong.
```

Comment:
Want to merge as is or is this a fix before merge?

Reply:
meant it to be a question. My logging shows that every example I have tried ... works. But it looks wrong to me and there is a Firebase crash with a dodgy regex. I'v e added logging, so we can see what it is.